src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/53.diff.sql

```diff
@@ -0,0 +1,1223 @@
+
+            IF NOT EXISTS (SELECT * FROM sys.indexes WHERE object_id = object_id('dbo.Tmp_QuantitySearchParam') AND name = 'IX_QuantitySearchParam_SearchParamId_QuantityCodeId_SingleValue')
+            CREATE NONCLUSTERED INDEX IX_QuantitySearchParam_SearchParamId_QuantityCodeId_SingleValue
+                ON dbo.Tmp_QuantitySearchParam(ResourceTypeId, SearchParamId, QuantityCodeId, SingleValue, ResourceSurrogateId)
+                INCLUDE(SystemId) WHERE IsHistory = 0
+                                        AND SingleValue IS NOT NULL
+                ON PartitionScheme_ResourceTypeId (ResourceTypeId);
+
+            IF NOT EXISTS (SELECT * FROM sys.indexes WHERE object_id = object_id('dbo.Tmp_QuantitySearchParam') AND name = 'IX_QuantitySearchParam_SearchParamId_QuantityCodeId_LowValue_HighValue')
+            CREATE NONCLUSTERED INDEX IX_QuantitySearchParam_SearchParamId_QuantityCodeId_LowValue_HighValue
+                ON dbo.Tmp_QuantitySearchParam(ResourceTypeId, SearchParamId, QuantityCodeId, LowValue, HighValue, ResourceSurrogateId)
+                INCLUDE(SystemId) WHERE IsHistory = 0
+                ON PartitionScheme_ResourceTypeId (ResourceTypeId);
+
+            IF NOT EXISTS (SELECT * FROM sys.indexes WHERE object_id = object_id('dbo.Tmp_QuantitySearchParam') AND name = 'IX_QuantitySearchParam_SearchParamId_QuantityCodeId_HighValue_LowValue')
+            CREATE NONCLUSTERED INDEX IX_QuantitySearchParam_SearchParamId_QuantityCodeId_HighValue_LowValue
+                ON dbo.Tmp_QuantitySearchParam(ResourceTypeId, SearchParamId, QuantityCodeId, HighValue, LowValue, ResourceSurrogateId)
+                INCLUDE(SystemId) WHERE IsHistory = 0
+                ON PartitionScheme_ResourceTypeId (ResourceTypeId);
+        END
+
+    IF object_id('dbo.Tmp_TokenNumberNumberCompositeSearchParam') IS NULL
+        BEGIN
+            CREATE TABLE dbo.Tmp_TokenNumberNumberCompositeSearchParam (
+                ResourceTypeId      SMALLINT        NOT NULL,
+                ResourceSurrogateId BIGINT          NOT NULL,
+                SearchParamId       SMALLINT        NOT NULL,
+                SystemId1           INT             NULL,
+                Code1               VARCHAR (256)   COLLATE Latin1_General_100_CS_AS NOT NULL,
+                SingleValue2        DECIMAL (36, 18) NULL,
+                LowValue2           DECIMAL (36, 18) NULL,
+                HighValue2          DECIMAL (36, 18) NULL,
+                SingleValue3        DECIMAL (36, 18) NULL,
+                LowValue3           DECIMAL (36, 18) NULL,
+                HighValue3          DECIMAL (36, 18) NULL,
+                HasRange            BIT             NOT NULL,
+                IsHistory           BIT             NOT NULL,
+                CodeOverflow1       VARCHAR (MAX)   COLLATE Latin1_General_100_CS_AS NULL
+            );
+            IF object_id('DF_Tmp_TokenNumberNumberCompositeSearchParam_IsHistory') IS NULL
```

Comment:
>IF object_id('DF_Tmp_TokenNumberNumberCompositeSearchParam_IsHistory') IS NULL [](http://example.com/codeflow?start=12&length=78)

Create index statements cannot be under table existence check. Please move them on the same level as table creates.

Reply:
moved to same level as table creates.